lib/Xmds/Soap4.php

```diff
@@ -271,6 +285,11 @@ public function RegisterDisplay($serverKey, $hardwareKey, $displayName, $clientT
         $display->clientType = $clientType;
         $display->clientVersion = $clientVersion;
         $display->clientCode = $clientCode;
+        $display->osVersion = $osVersion;
+        $display->osSdk = $osSdk;
+        $display->manufacturer = $manufacturer;
+        $display->brand = $brand;
+        $display->model = $model;
         //$display->operatingSystem = $operatingSystem;
```

Comment:
As per the above comment, this is where we'd parse out the existing parameter.

Reply:
Based on the latest [commit](https://github.com/xibosignage/xibo-cms/pull/2574/commits/48e7c9c0fab2c49afeb7846b0a06ac615c016a81), requesting re-review here as well, please. Thank you!